src/component/UserDetails.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import Table from './Table'
+import { Button, Stack } from '@mui/material'
+import { NavLink } from 'react-router-dom'
+
+function UserDetails() {
+  return (
+    <>
+    <Stack   >
+    
+    <NavLink
```

Comment:
what is the use of navlink here ?
when are using button itself, you can add onClick event on button